schema.sql

```diff
@@ -7,5 +7,13 @@ CREATE TABLE animals (
     escape_attempts integer,
     neutered boolean,
     weight_kg decimal,
-    species varchar(100)
+    species_id integer,
+    owners_id integer,
+    PRIMARY KEY (id),
+    CONSTRAINT species_id
+        FOREIGN KEY (species_id)
+            REFERENCES species (species_id),
+    CONSTRAINT owners_id
+        FOREIGN KEY (owners_id)
+            REFERENCES owners (owners_id)
```

Comment:
- [ ] According to the project requirement,  you should add a new table with the name `owners` with the following columns:

   ```sql
      CREATE TABLE owners (
      id int NOT NULL IDENTITY(1,1),
      full_name varchar(255),
      age int,
      PRIMARY KEY (id)
      );
   ```

- [ ] Then, you should also add a new table with the name `species` with the following columns:

   ```sql
      CREATE TABLE species (
      id int NOT NULL IDENTITY(1,1),
      name varchar(255),
      PRIMARY KEY (id)
      );
   ```

- [ ] And Make sure you modify the `animals` table to:

   * Remove column `species`
   * Add column `species_id` which is a foreign key referencing `species` table
   * Add column `owner_id` which is a foreign key referencing the `owners` table